importScripts/sampleRecords/media.source.json

```diff
@@ -0,0 +1,151 @@
+[{
+  "ObjectsImagesID": "10010",
```

Comment:
This image doesn't exist on the drive - I've sent a list of missing images to the client.